static/js/modules/columns.js

```diff
@@ -117,104 +115,99 @@ var committees = [
     }
   },
   {data: 'treasurer_name', className: 'min-desktop hide-panel'},
-  {data: 'state', className: 'min-desktop hide-panel', width: '80px'},
-  {data: 'party_full', className: 'min-desktop hide-panel'},
-  dateColumn({data: 'first_file_date', className: 'min-tablet hide-panel'}),
   {data: 'committee_type_full', className: 'min-tablet hide-panel'},
   {data: 'designation_full', className: 'min-tablet hide-panel'},
-  {data: 'organization_type_full', className: 'min-desktop hide-panel'},
+  dateColumn({data: 'first_file_date', className: 'min-tablet hide-panel column--med' }),
   modalTriggerColumn
 ];
 
 var communicationCosts = [
   {
     data: 'committee_name',
     orderable: false,
-    className: 'min-desktop',
+    className: 'all column--xl',
     render: renderCommitteeColumn,
   },
-  currencyColumn({data: 'transaction_amount', className: 'min-tablet'}),
-  supportOpposeColumn,
+  _.extend({}, supportOpposeColumn, {className: 'min-tablet hide-panel-tablet column--med'}),
   {
     data: 'candidate_name',
     orderable: false,
-    className: 'min-desktop hide-panel',
+    className: 'min-tablet hide-panel-tablet column--large',
     render: renderCandidateColumn
   },
-  dateColumn({data: 'transaction_date', className: 'min-tablet hide-panel-tablet'}),
+  currencyColumn({data: 'transaction_amount', className: 'min-tablet hide-panel column--med column--number'}),
+  dateColumn({data: 'transaction_date', className: 'min-tablet hide-panel column--med'}),
   modalTriggerColumn
 ];
 
 var disbursements = [
   {
-    data: 'recipient_name',
+    data: 'committee',
```

Comment:
just double checking the change `recipient_name` should be replaced by `committee`